### PR TITLE
Ltd 5446 update licences list information

### PR DESCRIPTION
**Pull request by kevincarrogan**
### Aim

Update the licence list to show the relevant good details.

I've also tidied up some of the other e2e tests around this to remove skipped e2e tests and the fixtures related to those tests.

[LTD-5446](https://uktrade.atlassian.net/browse/LTD-5446)

[LTD-5446]: https://uktrade.atlassian.net/browse/LTD-5446?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: exporter/templates/licences/licences.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h1 class="govuk-heading-l">{% block title %}{% lcs "licences.LicencesList.TITLE
 	{% include 'filters.html' %}
 
 	{% if data.results %}
-		<table class="govuk-table">
+		<table class="govuk-table licence-details-table" id="licence-details-table">
 			<thead class="govuk-table__head">
 				<tr class="govuk-table__row">
 					<th class="govuk-table__header" scope="col">{% lcs "licences.LicencesList.Table.REFERENCE_COLUMN" %}</th>
```
Comment (depsiatwal) [inline exporter/templates/licences/licences.html:40]: worth removing this legacy constants while we at it. ?

**Review by depsiatwal — APPROVED** [review]
just small comment but other wise looks good

File: unit_tests/conftest.py [hunk 2]
```diff
@@ -2616,3 +2618,11 @@ def _render_form(form, request=None):
         return rendered
 
     return _render_form
+
+
+@pytest.fixture()
+def beautiful_soup():
+    def _beautiful_soup(html):
```
Comment (Tllew) [inline unit_tests/conftest.py:2625]: this is nice

File: exporter/templates/licences/licences.html (hunk 1 above)
Comment (kevincarrogan) [reply exporter/templates/licences/licences.html:40]: I'm going to leave them for now as it's quite a bit of a bigger PR due to some of the other tidying up.

**Pull request merged**